docs/Ingress.md

```diff
@@ -53,7 +53,7 @@ For instructions on how to install and configure Istio for your specific infrast
 
 Most scenarios for Istio will require the configuration of a Gateway and a Virtual Service. Familiarize yourself with the [Istio Gateway](https://istio.io/latest/docs/reference/config/networking/gateway/) and [Istio Virtual Service ](https://istio.io/latest/docs/reference/config/networking/virtual-service/).
 
-### Configuring Ingress for Splunk Web
+### Configuring Ingress for Splunk Web and HEC
```

Comment:
I see we have updated the heading to include HEC, but the sub-points under the topic have no examples for HEC?

Reply:
Under the "Multiple Hosts Configuration" sub-point we reference HEC, 
`If your deployment has multiple hosts such as Search Heads and Cluster Master, use this example to configure Splunk Web access, and HTTP Event Collector port. Follow the steps here [HEC Documentation](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) to learn how to create a HEC token and how to send data using HTTP.`
perhaps I could update that sub point heading to Multiple Hosts and HEC Configuration? Or perhaps we could include a solely HEC example with a standalone

Reply:
Yes, at least an update to the existing Heading will be good!